stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -296,8 +314,26 @@ spec:
                   /bin/mongodb_exporter
                   --mongodb.uri {{ template "mongodb-replicaset.connection-string" . }}
                   --test
-            initialDelaySeconds: 30
-            periodSeconds: 10
+            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+{{- if semverCompare ">=1.16-0" .Capabilities.KubeVersion.GitVersion -}}
+          startupProbe:
+            exec:
+              command:
+                - mongo
```

Comment:
This shouldn't test mongo directly but run the same test as the readiness probe. The readiness probe already tests the mongo connection itself.